src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -876,6 +876,360 @@ public CompilerLoweringPreserveAttribute() { }
             }
             """;
 
+        #region A string containing expression-tree dumping utilities
+        protected const string ExpressionTestLibrary = """
```

Comment:
Consider making this `static readonly` instead of `const` to avoid duplicating the string in each DLL. See https://github.com/dotnet/roslyn/pull/74307